tests/tx_builder.rs

```diff
@@ -0,0 +1,184 @@
+use anyhow::{anyhow, Result};
+use bip39::Mnemonic;
+use blockfrost::{BlockfrostAPI, Pagination};
+use blockfrost_openapi::models::{AddressUtxoContentInner, EpochParamContent};
+use cardano_serialization_lib::{
+    hash_transaction, make_vkey_witness, Address, BaseAddress, BigNum, Bip32PrivateKey,
+    CoinSelectionStrategyCIP2, Credential, LinearFee, NetworkId, PrivateKey, Transaction,
+    TransactionBody, TransactionBuilder, TransactionBuilderConfigBuilder, TransactionHash,
+    TransactionInput, TransactionOutput, TransactionUnspentOutput, TransactionUnspentOutputs,
+    TransactionWitnessSet, Vkeywitnesses,
+};
+
+pub async fn build_tx(blockfrost_client: &BlockfrostAPI) -> Result<Transaction> {
+    let bip32_prv_key = mnemonic_to_private_key(
+        "bright despair immune pause column saddle legal minimum erode thank silver ordinary pet next symptom second grow chapter fiber donate humble syrup glad early",
```

Comment:
Similar question – isn’t this a secret? Someone might restore this wallet, "steal" our Preview funds, and sort of DoS our integration tests. Not a great tragedy, but still a nuisance. :sweat_smile:

Reply:
(And devs can have the key in `.env` for local runs – we already customize `NODE_SOCKET_PATH` for integration tests.)

Reply:
An explanation of why this is okay [is in a thread above](https://github.com/blockfrost/blockfrost-platform/pull/189#discussion_r1967411567) – fair enough!